web/modules/custom/collection/src/Entity/GlossaryTermInterface.php

```diff
@@ -0,0 +1,41 @@
+<?php
+
+declare(strict_types = 1);
+
+namespace Drupal\collection\Entity;
+
+use Drupal\node\NodeInterface;
+
+/**
+ * Interface for glossary terms in Joinup.
+ */
+interface GlossaryTermInterface extends NodeInterface {
```

Comment:
Shouldn’t GlossaryTermInterface extend also GroupContentInterface?

Reply:
I only added the methods strictly needed in scope of this PR, but yes we should add this when the need arises.